### DISCUSSION TITLE
Short white flash before the page loads.

### DISCUSSION THREAD

**Discussion by skrik2** [requarks/wiki#5938] [Error / Bug Report] [answered]
I use Edge and chrome.
Hopefully it can be optimized.

**Comment by zackcain** [comment]
I'm getting the exact same issue right now, is there a solution? I'm using the Dark Theme and the white flash between page loads is really jarring

**Reply by NGPixel** [reply, marked as answer]
This will be fixed in the upcoming v3 release.
